### ISSUE TITLE
Disabling indexing for a section seems to take no effect

### ISSUE THREAD

**Issue by RosanaRufer** [nystudio107/craft-seomatic#138]
Hi,

I have disabled indexing for three sections on a site but they  are still included in the sitemap.xml and Google Search Console is reporting 404s on those disabled sections.

**This is what I did for these sections:**

<img width="946" alt="image" src="https://user-images.githubusercontent.com/4906291/41238891-fde60990-6d8e-11e8-997e-50882725f788.png">

**The Content SEO panel looks like this:**

<img width="773" alt="image" src="https://user-images.githubusercontent.com/4906291/41238870-f0817708-6d8e-11e8-97d4-e0fb552fa3a6.png">

But they are still in the sitemap.xml Is this a bug or a new newbism of mine?

Thank you

**Comment by khalwat** [comment]
What version of SEOmatic are you using?

**Comment by khalwat** [comment]
So it is checking for whether or not to include a given sitemap index in the primary `sitemap.xml`:

```
            /** @var  $metaBundle MetaBundle */
            foreach ($metaBundles as $metaBundle) {
                if (\in_array($metaBundle->sourceSiteId, $groupSiteIds, false)
                    && $metaBundle->metaSitemapVars->sitemapUrls) {
```

I wonder if this could be a caching issue?

**Comment by khalwat** [comment]
I just did a test where I disabled a section's URLs and they did indeed disappear from the `sitemap.xml`. Is this a multi-locale site? You'd need to disable them for each locale if so.

**Comment by khalwat** [comment]
Before:

![google chromescreensnapz1213](https://user-images.githubusercontent.com/7570798/41261116-483cc9de-6da8-11e8-9420-82a60a813d71.png)

...then after disabling Sitemap URLs for the 1st Homepage section locale:

![google chromescreensnapz1214](https://user-images.githubusercontent.com/7570798/41261135-5671b258-6da8-11e8-972a-748e106ef083.png)

**Comment by RosanaRufer** [comment]
It got fixed after upgrading, I was working with 3.0.18 and since I didn't see any mention to 'sitemap' on the changelog I did not try upgrading until today. Thank you again and sorry for the trouble.
